src/main/java/organice/model/AddressBook.java

```diff
@@ -68,13 +71,29 @@ public boolean hasPerson(Person person) {
     }
 
     /**
-     * Returns true if a doctor with the same nric as {@code doctorInCharge} exists in the address book.
+     * Returns true if a doctor with the same NRIC as {@code doctorInCharge} exists in the address book.
      */
     public boolean hasDoctor(Nric doctor) {
         requireNonNull(doctor);
         return persons.containsDoctor(doctor);
     }
 
+    /**
+     * Returns true if a patient with the same NRIC as {@code patient} exists in the address book.
+     */
+    public boolean hasPatient(Nric patient) {
```

Comment:
Do note that my previous PR already has this code, so you may need to tweak changes when you merge my PR in.

Reply:
resolved the conflict